src/main/java/org/gridsuite/directory/server/repository/DirectoryElementEntity.java

```diff
@@ -38,8 +40,8 @@
     @Column(name = "type", length = 30, nullable = false)
     private String type;
 
-    @Column(name = "isPrivate", nullable = false)
-    private boolean isPrivate;
+    @Column(name = "isPrivate")
+    private Boolean isPrivate;
```

Comment:
Is it on purpose that you have a Boolean and not boolean?

Reply:
I want to be able to pass a null as value for the elements